src/wxvx/workflow.py

```diff
@@ -505,6 +535,20 @@ def _enforce_point_baseline_type(c: Config, taskname: str):
         raise WXVXError(msg % taskname)
 
 
+def _message_exists(path: Path, var: Var) -> bool:
+    idx = ec.codes_index_read(str(path))
+    ec.codes_index_select(idx, "shortName", var.name)
+    ec.codes_index_select(idx, "typeOfLevel", var.level_type)
+    ec.codes_index_select(idx, "level", int(var.level) if var.level else 0)
```

Comment:
I don't feel strongly, and it's more lines but less code/repetition:
```suggestion
    for k, v in [
        ("shortName", var.name),
        ("typeOfLevel", var.level_type),
        ("level", int(var.level) if var.level else 0),
    ]:
        ec.codes_index_select(idx, k, v)
```